client/components/Map/Map.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import {
+  atom,
+  selector,
+  useRecoilState,
+  useRecoilValue,
+  useSetRecoilState,
+} from 'recoil';
+import MapView, { Marker, PROVIDER_GOOGLE } from 'react-native-maps';
+import { StyleSheet, Dimensions } from 'react-native';
+
+import axios from 'axios';
+
+import userLocationState from '../../state/atoms/userLocation';
+import currentMapView from '../../state/atoms/currentMapView';
+const { GOOGLE_MAPS_API_KEY_IOS } = process.env;
+
+const { height, width } = Dimensions.get('window');
+
+const Map = () => {
+  const currentView = useRecoilValue(currentMapView);
+  const userLocation = useRecoilValue(userLocationState);
```

Comment:
I think to use this atom `useRecoilState(currentMapView)` is needed, but if this works then no big deal.